projects/media-viewer/src/lib/media-viewer/toolbar/search-bar/search-bar.component.ts

```diff
@@ -63,4 +74,10 @@ export class SearchBarComponent {
         : 'Phrase not found';
     }
   }
+
+  public onInputKeyPress(e: KeyboardEvent) {
```

Comment:
is a HostListener needed with onWindowKeyDown and not here because it's dealing with control keys and not normal input keys?

Reply:
doesn't seem to be